astrokat/simulate.py

```diff
@@ -135,6 +136,7 @@ def __iter__(self):
         raise StopIteration
 
     def __exit__(self, type, value, traceback):
+        # TODO: what is self.track_ used for again?
         if self.track_:
```

Comment:
No longer creating this variable in `__init__`, so can we be sure it will even exist?

Reply:
Incomplete implementation, only realised the inconsistency when I wrote the planning integration scripts

This must be completed for multiple observation loops, will be addressed in separate devel branch